imports/startup/lib/dj.js

```diff
@@ -25,9 +27,11 @@ DJ.allow('collections.reviews.showUpdate', true);
 DJ.allow('collections.reviews.showRemove', true);
 
 DJ.deny('collections.reviews.update', (userId, doc, fields, modifier) =>
-  (!_.contains(fields, 'approved')));
```

Comment:
It appears that I had misinterpreted the functionality of `deny()`. The reason is trivial and left as an exercise to the reader.